app/components/appMainModule/committeePageModule/state.js

```diff
@@ -15,23 +15,8 @@ module.exports = function($stateProvider) {
     template: '<ui-view></ui-view>',
     controller: 'committeePageController',
     resolve: {
-      committee: function($stateParams, $q) {
-        return $q.resolve({
-          committee_id: 1234,
-          name: 'Americans for Liberty',
-          contribution_by_type: {
-            unitemized: 2916394,
-            self_funded: 512554,
-            political_party: 6426112,
-            individual: 11134547,
-            recipient_committee: 986229
-          },
-          contribution_by_area: {
-            inside_location: 0.56,
-            inside_state: 0.38,
-            outside_state: 0.06
-          }
-        });
+      committee: function($stateParams, disclosureApi) {
+        return disclosureApi.committee.get();
```

Comment:
You need to pass the committee_id parameter because it's required.

```javascript
return disclosureApi.committee.get({committee_id: $stateParams.committee_id});
```